devscripts/prepare_manpage.py

```diff
@@ -6,7 +6,33 @@
 import re
 
 ROOT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-README_FILE = os.path.join(ROOT_DIR, 'README.md')
+README_FILE = os.path.join(ROOT_DIR, sys.argv[1])
+
+
+def filter_options(readme):
+    ret = ''
```

Comment:
I don't know if there's an actual performance gain, but it's common to use a list:
```
ret = []
for x in ...:
    ret.append(newstring)
return ''".join(ret)
```